src/main/webapp/index.html

```diff
@@ -20,7 +20,12 @@
         <label for="actions-radio" id="actions-label" class="menu-label">Action</label>
       </div>
       <div id="selected-option">
-        <div id="slogan"><p>Slogan</p></div>
+        <!--<div id="slogan"><p>Slogan</p></div>-->
+        <form action="/keyword" method="POST">
+          <textarea name="keyword-sentence" style="resize: none; width: 600px; height: 350px;" placeholder="Enter your tweet or interests here"></textarea>
```

Comment:
You may want to add an external css style sheet.

Reply:
I will put this in index.css instead.